test/test_sparse.py

```diff
@@ -738,7 +738,7 @@ def test_cross_device(x1, x2):
         self.assertEqual(None, x1.grad)
 
     @onlyCUDA
-    def test_cuda_empty(self, _):
+    def test_cuda_empty(self):
```

Comment:
These tests should accept a device argument and use it. The argument is not a device type but an actual device. Like "cuda:0" or "cuda:1", but these tests are ignoring the test device in favor of "cuda:0"

Reply:
This has been fixed in the previous parametrization PR- once I rebase the change will show up here too